resolwe/process/fields.py

```diff
@@ -512,7 +515,13 @@ def __init__(self, data_id, field, cache):
 
             field_type = field_descriptor["type"].rstrip(":")
 
-            if field_type.startswith("list:"):
+            if field_type.startswith("list:data"):
```

Comment:
Add colon at the end.

Reply:
Check two lines above. :-)

Reply:
> I don't like this solution, because it is not 100% correct :)

Why do you mean by "it is not 100% correct"?

Reply:
If someone uses type `list:dataid:`, it would match this condition instead of `elif`.

Reply:
How about `if field_descriptor["type"].startswith("list:data:"):`?

Reply:
Then it won't match fields of type `list:data:`, because final colon is striped in the line above.

Reply:
It is not stripped in `field_descriptor["type"]`

Reply:
Sorry, i haven't seen this change :) (`field_type` to `field_descriptor["type"]`).

Yes, that would also work, but you have to add a test for this or at least a comment.